accepted/future-releases/unquoted-imports/feature-specification.md

```diff
@@ -353,11 +353,11 @@ parse while looking like a single opaque "unquoted string" to users and tools.
 
 The normative stuff starts now. Here is the proposal:
 
-We add a new rule and hang it off the existing `uri` rule already used by import
-and export directives:
+We add a new rule and hang it off the existing `configurableUri` rule already
+used by import and export directives:
 
 ```
-uri               ::= stringLiteral | packagePath
+configurableUri   ::= ( stringLiteral | packagePath ) configurationUri*
```

Comment:
This will only fix it for a brief time since "enhanced parts" allows `part` directives to be configurable.
(If enhanced parts won't lands first.)

Maybe just use the existing grammar, and say that "It's a compile-time error if a `part` or `part of` directive contains a `uri` which is a `packagePath`."

Then the parser can assume that `part`/`part of` must be followed by a string literal, and it's all error recovery if it isn't. (If anyone uses `part mypackagename/src/foo_part;` we can probably tell then what is wrong.)

Reply:
I like capturing as much of the syntax in the grammar as reasonably possible, so I went ahead and grammarified it. But I added a note about the connection to parts-with-imports.